auacm/angular/src/app/blog-list/blog-list.component.ts

```diff
@@ -38,7 +38,7 @@ export class BlogListComponent implements OnInit {
       const tempPipe = new LimitWordsPipe();
       this.blogPosts = blogs;
       for (let i = 0; i < this.blogPosts.length; i++) {
-        if (tempPipe.transform(this.blogPosts[i].body, 200).trim().length < this.blogPosts[i].body.trim().length) {
+        if (tempPipe.transform(this.blogPosts[i].body, 200).trim().length < this.converter.makeHtml(this.blogPosts[i].body.trim()).length) {
```

Comment:
Can we make this 200 be a constant or something? This is a bit of magic number, and this way the HTML and component won't be out of sync.